dev/VSIX/Extension/Cs/Common/VSPackage.resx

```diff
@@ -118,31 +118,31 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="1001" xml:space="preserve">
-    <value>Blank Window (WinUI 3 in Desktop)</value>
+    <value>Blank Window</value>
```

Comment:
Would it be better to append its extension like `Blank Window (.xaml/.cs)` or `Resource Dictionary (.xaml)`.

Reply:
Good idea - do other resources templates do this?

Reply:
This is not common for item templates for other frameworks, and adds little value?